### PR TITLE
Add example Emoji Bar Chart

### PR DESCRIPTION
**Pull request by chanwutk**
The example is inspired by @jakevdp.
visualization 1

### CODE REVIEW

File: examples/specs/isotype_bar_chart_emoji.vl.json [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+      {"country": "United States", "animal": "sheep"},
+      {"country": "United States", "animal": "sheep"},
+      {"country": "United States", "animal": "sheep"},
+      {"country": "United States", "animal": "sheep"},
+      {"country": "United States", "animal": "sheep"}
+    ]
+  },
+  "transform": [
+    {
+      "calculate": "datum.animal==='cattle' ? '🐄' : datum.animal==='pigs' ? '🐖' : '🐏'",
```
Comment (domoritz) [inline examples/specs/isotype_bar_chart_emoji.vl.json]: Can we use a scale domain instead?

**Comment by domoritz** [comment]
Why does this not work?

```json
{
  "$schema": "https://vega.github.io/schema/vega-lite/v2.json",
  "config": {"view": {"stroke": ""}},
  "width": 800,
  "height": 200,
  "data": {
    "values": [
      {"country": "Great Britain", "animal": "pigs"},
      {"country": "Great Britain", "animal": "pigs"},
      {"country": "Great Britain", "animal": "cattle"},
      {"country": "Great Britain", "animal": "cattle"},
      {"country": "Great Britain", "animal": "cattle"},
      {"country": "Great Britain", "animal": "sheep"},
      {"country": "Great Britain", "animal": "sheep"},
      {"country": "Great Britain", "animal": "sheep"},
      {"country": "Great Britain", "animal": "sheep"},
      {"country": "Great Britain", "animal": "sheep"},
      {"country": "Great Britain", "animal": "sheep"},
      {"country": "Great Britain", "animal": "sheep"},
      {"country": "Great Britain", "animal": "sheep"},
      {"country": "Great Britain", "animal": "sheep"},
      {"country": "Great Britain", "animal": "sheep"},
      {"country": "United States", "animal": "pigs"},
      {"country": "United States", "animal": "pigs"},
      {"country": "United States", "animal": "pigs"},
      {"country": "United States", "animal": "pigs"},
      {"country": "United States", "animal": "pigs"},
      {"country": "United States", "animal": "pigs"},
      {"country": "United States", "animal": "cattle"},
      {"country": "United States", "animal": "cattle"},
      {"country": "United States", "animal": "cattle"},
      {"country": "United States", "animal": "cattle"},
      {"country": "United States", "animal": "cattle"},
      {"country": "United States", "animal": "cattle"},
      {"country": "United States", "animal": "cattle"},
      {"country": "United States", "animal": "cattle"},
      {"country": "United States", "animal": "cattle"},
      {"country": "United States", "animal": "sheep"},
      {"country": "United States", "animal": "sheep"},
      {"country": "United States", "animal": "sheep"},
      {"country": "United States", "animal": "sheep"},
      {"country": "United States", "animal": "sheep"},
      {"country": "United States", "animal": "sheep"},
      {"country": "United States", "animal": "sheep"}
    ]
  },
  "transform": [
    {"window": [{"op": "rank", "as": "rank"}], "groupby": ["country", "animal"]}
  ],
  "mark": {"type": "text", "baseline": "middle"},
  "encoding": {
    "x": {"field": "rank", "type": "ordinal", "axis": null},
    "y": {"field": "animal", "type": "nominal", "axis": null, "sort": null},
    "row": {"field": "country", "type": "nominal", "header": {"title": ""}},
    "text": {
      "field": "animal", "type": "nominal",
      "scale": {
        "domain": ["cattle", "pigs", "sheep"],
        "range": ["🐄", "🐖", "🐏"]
      }
    },
    "size": {"value": 65}
  }
}
```

**Comment by domoritz** [comment]
Let's squash this when travis has finished.

**Comment by chanwutk** [comment]
Should we build the spec?

**Comment by domoritz** [comment]
Let Travis do it.

**Pull request merged**